Yoast/wordpress-seo
# P2 477 content on import and export page missing

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a bug where there was no content on the import- and export tool, file editor tool and bulk editor tool pages.

## Relevant technical choices:

* Added an extra check if we are on the _main_ tools page, instead of a specific tools page, in the `Yoast_Tools_Page_Conditional`.
  * This conditional is only used in the indexing integration.
  * However, this may break backwards compatibility and the name does not 100% apply (we make the check stricter).
   * It may be better to add a new `Yoast_Main_Tools_Page_Conditional`, and to deprecate the old one?
* Added an extra safety check in the `indexation.js` script, to check if the DOM-container in which we render exists.

## Test instructions

This PR can be tested by following these steps:

* Navigate to SEO - Tools and click on _Import and Export_.
* See that the content of these pages is **not** missing.
* Do the same for the _File editor_ and the _Bulk editor_ on the tools page.

## UI changes

## Documentation

## Quality assurance


Fixes #
